diff --git a/node_modules/typedoc-plugin-markdown/dist/subthemes/docusaurus2/theme.js b/node_modules/typedoc-plugin-markdown/dist/subthemes/docusaurus2/theme.js
index a53ab88..e67d2a5 100644
--- a/node_modules/typedoc-plugin-markdown/dist/subthemes/docusaurus2/theme.js
+++ b/node_modules/typedoc-plugin-markdown/dist/subthemes/docusaurus2/theme.js
@@ -14,12 +14,16 @@ class Docusaurus2Theme extends theme_1.default {
         this.listenTo(renderer, events_1.RendererEvent.END, this.onRendererEnd, 1024);
     }
     onRendererEnd(renderer) {
-        const docusarusRoot = this.findDocusaurus2Root(renderer.outputDirectory);
-        if (docusarusRoot === null) {
-            this.application.logger.warn(`[typedoc-markdown-plugin] ${this.sidebarName} not written as could not locate docusaurus root directory. In order to to implemnent ${this.sidebarName} functionality, the output directory must be a child of a 'docs' directory.`);
-            return;
+        if (!this.application.options.getValue('skipSidebar')) {
+            const docusarusRoot = this.findDocusaurus2Root(renderer.outputDirectory);
+            if (docusarusRoot === null) {
+                this.application.logger.warn(
+                    `[typedoc-markdown-plugin] ${this.sidebarName} not written as could not locate docusaurus root directory. In order to to implemnent ${this.sidebarName} functionality, the output directory must be a child of a 'docs' directory.`,
+                );
+                return;
+            }
+            this.writeSideBar(renderer, docusarusRoot);
         }
-        this.writeSideBar(renderer, docusarusRoot);
     }
     writeSideBar(renderer, docusarusRoot) {
         const childDirectory = renderer.outputDirectory.split(docusarusRoot + 'docs/')[1];
